smdebug/tensorflow/keras.py

```diff
@@ -366,6 +394,106 @@ def _add_metric(self, metric_name, metric_value: tf.Tensor = None):
             coll.set_tensor_ref(TensorRef.from_non_graph_var(metric_name))
         self.tensor_to_collections[metric_name] = {coll}
 
+    def save_tensor(self, tensor_name, tensor_value, collections_to_write=None):
+        if (
+            not ((isinstance(tensor_value, tf.Tensor)) and hasattr(tensor_value, "numpy"))
+        ) or self._is_not_supported():
+            return
+        if collections_to_write is None:
+            collections_to_write = "default"
```

Comment:
nit: the default value on L397 can be "default" instead of this check?

Reply:
Nice! Thanks!